package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "cucumber-boilerplate",
+  "version": "1.3.2",
+  "description": "Boilerplate project to run WebdriverIO tests with Cucumber",
+  "homepage": "https://github.com/webdriverio/cucumber-boilerplate",
+  "scripts": {
+    "clean": "read -p \"Are you sure you want to clean the project? [y/n] \" REPLY; if [[ $REPLY =~ ^[Yy]$ ]]; then rm -R .git .github demo-app test .codeclimate.yml .travis.yml jest.json wdio.BUILD.conf.js src/features/**; fi",
+    "local-webserver": "forever -a --uid \"my-wdio-local-webserver\" start -s ./node_modules/.bin/http-server ./demo-app -s",
+    "list-local-webserver": "forever list",
+    "list-logs-local-webserver": "forever logs",
+    "logs-local-webserver": "forever logs \"my-wdio-local-webserver\"",
+    "stop-local-webserver": "forever stop my-wdio-local-webserver",
+    "stopall-local-webserver": "forever stopall",
+    "test": "npm run test:validate && npm run test:unit && npm run test:features",
+    "test:features": "npm run local-webserver & (wdio wdio.BUILD.conf.js; wdio_ret=$?; forever stopall; exit $wdio_ret)",
+    "test:unit": "jest --config=jest.json",
+    "test:validate": "eslint 'src/**/*.js' 'test/**/*.js!(setup.js)'",
+    "e2etests:local": "yarn run wdio ",
+    "e2etests:pipeline": "yarn run wdio wdio.PIPELINE.conf.js ",
+    "e2etests:dev": "yarn run wdio wdio.DEV.conf.js",
+    "e2etests:pre": "yarn run wdio wdio.PRE.conf.js",
+    "e2etests:pro": "yarn run wdio wdio.PRO.conf.js"
```

Comment:
Do we really want all these scripts in the package? I mean, these belong to each repo, but not to the package itself.

Reply:
No, you are right. New commit done. I have included dependencies that maybe we can change in future.